policy/controller.go

```diff
@@ -30,7 +30,7 @@ func sortedKeys[K constraints.Ordered, V any](m map[K]V) []K {
 
 // TestVariations models the variations of the test matrix in
 // release.yml:api-tests. Each key is a row in the matrix.
-type TestVariations map[string][]string
+type TestVariations map[string]interface{}
```

Comment:
map[string]interface invalidates the type guarantees that we need make sure that the correct type is passed to the matrix.
Perhaps the excludes can be a new type?

Reply:
yes in that case we need a new type, I was between that or using interface. Maybe we can build a struct that contains the 2 types so it's easier to pass between functions or call methods?